javascript/javascript1/week1/Age-ify.js

```diff
@@ -0,0 +1,4 @@
+const ageOfBirth = 1970;
+const futureYear = 2050;
+let age = futureYear - ageOfBirth; 
+console.log ("l will be " + age + " years old in "  + futureYear);
```

Comment:
Great work! That is exactly how you could solve it. Another way would be to use what is called string interpolation:

``I will be ${ age } years old in ${ futureYear }``